decidim-proposals/app/packs/stylesheets/decidim/proposals/proposals.scss

```
@@ -103,11 +103,211 @@
+        }
+      }
+
+      .card__proposals-votes {
+        @apply flex md:flex-col lg:flex-row bg-[#F5F5F5] justify-around md:flex-[0.6] lg:flex-[0.8] items-center md:items-stretch lg:items-center lg:min-w-[196px] h-[68px] md:h-auto lg:h-[68px];
+
+        button {
+          @apply bg-[var(--secondary)];
+
+          &[disabled] {
+            @apply opacity-50 text-white border-current cursor-not-allowed;
+          }
+        }
+
+        .with_votes_limit {
+          @apply flex flex-col p-0 md:p-2 lg:p-0 md:flex-row md:flex-wrap lg:flex-col lg:flex-nowrap flex-1 md:flex-none lg:flex-1 justify-center;
+
+          .proposals__aside-progress {
+            @apply flex flex-col w-[80%] items-center md:order-1 lg:order-none md:flex-[1_1_100%] lg:flex-none;
+          }
+
+          .progress-bar__units {
+            @apply md:order-none;
+          }
+
+          .progress-bar__number {
+            span {
+              @apply text-sm;
+            }
+          }
+        }
+
+        .without_votes_limit {
+          @apply flex flex-row flex-1 lg:flex-1 justify-center md:flex-none;
+
+          .progress-bar__number {
+            span {
+              @apply text-xl;
+            }
+          }
+        }
+
+        .progress-bar {
+          &__number {
+            @apply flex justify-center items-center mx-1;
+
+            span {
+              @apply leading-6 font-semibold #{!important};
+            }
+          }
+
+          &__units {
+            @apply text-sm leading-[22px] mx-1;
+          }
+        }
+
+        .proposal_votes_container {
+          @apply flex-1 md:w-full md:flex-none p-2 lg:p-0 lg:m-3 lg:w-[164px];
+
+          button {
+            @apply text-sm leading-[18px] font-semibold py-1;
+          }
+        }
+      }
+
+      .votes_hidden {
+        @apply md:flex-none;
+
+        .proposal_votes_container {
+          @apply md:flex-none md:w-[130px] lg:w-[164px] lg:p-0;
+        }
+      }
+    }
+  }
+
+  &__grid-grid {
+    @apply md:gap-4;
+
+    .card__proposals-item {
+      @apply flex flex-col justify-between border-solid border-[#D3D5D9] border rounded;
+
+      .card__grid {
+        @apply mb-2.5 md:mb-0 flex-1 justify-between;
+
+        box-shadow: none !important;
+
+        &-text {
+          @apply justify-end lg:gap-y-8;
+        }
+
+        &-metadata {
+          @apply mt-0 md:flex md:flex-row;
+
+          > :first-child {
+            flex: 1 !important;
+          }
+
+          > :not(:first-child) {
+            @apply border-r border-[#D3D5D9];
+          }
+
+          & > div {
+            @apply px-2 md:px-2;
+          }
+        }
+      }
+
+      .card__proposals-votes {
+        @apply flex bg-[#F5F5F5EE] justify-around items-center px-4 h-[68px];
+
+        button {
+          @apply bg-[var(--secondary)];
+
+          &[disabled] {
+            @apply opacity-50 text-white border-current cursor-not-allowed;
+          }
+        }
+
+        .with_votes_limit {
```

I think this should be a BEM modifier, so we should add a namespace here
```suggestion
        &-with_votes_limit {
```

---

changed to .card__proposals-votes-limited

---

done in [dd839af](https://github.com/decidim/decidim/pull/13883/commits/dd839af40bf2d480679578a5583a79078ff9ad0f)